### DISCUSSION TITLE
remote: Invalid username or password

### DISCUSSION THREAD

**Discussion by DennesTorres** [community/community#4743] [Codespaces] [answered]
One of my codespaces is resulting in this message when I try to make a git pull.  I'm not sure how to fix this problem in codespaces.

git config -l show the configuration values, but everything seems fine. I have not played directly with these values.

I already tried in different browsers, it's always the same result.

**Comment by tanmayeekamath** [comment]
Hey @DennesTorres 
Can you please confirm if you're trying to access a different repo from within a Codespace with git pull (other than the one it was created from)?

**Comment by DennesTorres** [comment]
Hi, Tanmayee,

No, I was not.

After I stopped using that codespace for one day or two, the error
disappeared, so I can't reproduce it anymore. However, if there is a
solution, it would be interesting to know in case it happens again.

Kind regards,

Dennes

Em seg., 26 de jul. de 2021 às 21:00, Tanmayee Kamath <
***@***.***> escreveu:

> Hey @DennesTorres <https://github.com/DennesTorres>
> Can you please confirm if you're trying to access a different repo from
> within a Codespace with git pull (other than the one it was created from)?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/github/feedback/discussions/4743#discussioncomment-1054118>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEWJZNU7FCHHGN2HQJAS7KTTZWWDLANCNFSM5AYGO5GQ>
> .
>


-- 

Kind Regards,

Dennes Torres
Data Platform MVP
MCT, MCSE (Data e BI), MCSD
MMDPUG PASS Chapter Leader
devSQL PASS Chapter Leader

**Comment by tanmayeekamath** [comment, marked as answer]
Thanks for confirming, @DennesTorres! 
We didn't have any known issues upon checking for this. I'm glad that you are not seeing the error anymore. Please reach out if this happens again.
